deployment/helm/README.md

```diff
@@ -0,0 +1,166 @@
+# EdgeX Foundry on Kubernetes
+
+A [Helm](https://helm.sh/) chart to easily deploy the EdgeX IoT project on Kubernetes.
+Based on EdgeX [Jakarta](https://github.com/edgexfoundry/edgex-compose/tree/jakarta) version.
+
+## Prerequisites
+
+- Kubernetes cluster 1.10+
+- [Helm](https://helm.sh/) 3.7.0+
+
+## Installation
+
+Install the EdgeX helm chart with a release name edgex-jakarta
+
+```bash
+$ git clone https://github.com/lenny-intel/edgex-helm
+$ cd edgex-helm
+$ git checkout jakarta
+$ kubectl create namespace edgex
+$ helm install edgex-jakarta -n edgex .
+```
+
+## Uninstallation
+
+```bash
+helm uninstall edgex-jakarta -n edgex
+```
+
+## Test EdgeX
+
+EdgeX on kubernetes using NodePort type to expose services by default,You can use ping command to test whether the EdgeX services start successfully.
+
+The ping command format:
+```bash
+http://<ExternalIP>:<ExposedPort>/api/v2/ping
+
+```
+For example, the edgex-core-data ping command format:
+
+```bash
+curl http://localhost:59880/api/v2/ping
+```
+
+
+## Access EdgeX UI
+
+With a modern browser, navigate to http://\<ExternalIP\>:30400.
```

Comment:
Just to confirm, this is the static external NodePort that will be used for this?

Reply:
Probably need to check all the port #'s.  All the custom nodeports were replaced with hostPort used by upstream EdgeX.  Primary motivation for this is that no validation has been done to make sure that EdgeX services run in a load-balanced configuration.  Once a service has been fully validated to run in HA mode we can put it back on a nodeport.